.github/workflows/release.yml

```diff
@@ -127,7 +127,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           CGO_ENABLED: 1
-          GO111MODULE: off
+          GO111MODULE: on
```

Comment:
This change will propagate to https://github.com/TykTechnologies/gromit/blob/master/testdata/policies/repos.yaml#L31 which does not seem right. I've asked @ilijabojanovic to add fix versions to the jira. This might need some update in the policy as this should _not_ propagate to -4-lts. Since 4.1 _and_ 4.2 are already released, this should _not_ go there, right?

Reply:
Yes,  the templates itself should probably be changed to be version specific regarding the module var, as older version will require this. ( https://github.com/TykTechnologies/gromit/blob/1cd7753da3d02c5fb07b6c8a3d7ec7c5fe1677b6/policy/templates/releng/.github/workflows/release.yml.d/goreleaser.tmpl#L116 ). 
For the propagation part - Can't we just close the auto generated PRs against the branches that don't require the changes?

Reply:
Closing the PR would work. Until the next PR :cry:

Reply:
Ah yes! :man_facepalming: